docs/Integration-with-CxSCA.md

```diff
@@ -179,4 +180,16 @@ In order to open SCA security tickets, set the bug tracker in cxflow app.yml fil
 In order to change the default CxFlow SCA scan behaviour and to perform a SCA ZIP scan, the next configuration property should be added underneath the sca configuration section:
 ```
 enabledZipScan: true
-```
\ No newline at end of file
+```
+
+## <a name="scaProjectTeamAssignment">SCA project team assignment</a>
+SCA project team assignment with CxFlow is performing on the SCA project creation stage. In order to set a project team, the next configuration property should be added underneath the sca configuration section:
+```
+teamForNewProjects: /team
```

Comment:
I guess we should use the 'team-for-new-projects' format here to conform to the rest of properties.

Reply:
Will be fixed